exploration/beauty-contest.md

```diff
@@ -10,11 +10,14 @@ Please dispute factual errors in this. The table uses multiple `-` when an optio
 | Option | Description                                                    | Doesn’t Nest {} | Doesn’t Need More Escapes | Doesn’t Require Quoted Pattern | Allows Unquoted Literal Operands | Counted {} works | Adds no sigils |
 | :----- | :------------------------------------------------------------- | :-------------- | :------------------------ | :----------------------------- | :------------------------------- | :--------------- | :------------- |
 | 0      | Current syntax, starts in code mode                            | -               | +                         | --                             | +                                | -                | +              |
+| 0a     | Hybrid syntax, starts in text mode                             | -               | +                         | -                              | +                                | -                | +              |
```

Comment:
As mentioned in slack, I think this should be 2a instead. 0 is start in code mode. This is a start-in-text-mode option

Reply:
I chose to call it `0a` because in my mind it's our current syntax extended with the "simple message" mode. But I agree it's closely related to `2` in terms of how they transition between modes.